_posts/newsletters/2020-11-30-newsletter.md

```diff
@@ -0,0 +1,115 @@
+
+<a name="sc"></a>
+# **Meet the Steering Committee Candidates**
+
+As [shared](https://us-rse.org/2020-06-29-sc-transition/) this summer, we're holding an election for four seats on the Steering Committee in December. Read about the candidates on the [website](https://us-rse.org/2020-11-23-sc-candidates/) and hear from them during the [AGM](https://us-rse.org/events/2020/2020-12-AGM), and ask any questions you might have for them on the [#election](https://usrse.slack.com/archives/C01BC66Q16E) channel on Slack.  You'll be asked to rank order the six candidates for the steering committee during the election, which runs December 13-19.  Good luck to all of the candidates, and thank you for volunteering!
+
+<a name="dei"></a>
+# **US-RSE Community Call on DEI**
+
+The October Community Call covered the topic "Diversity, Equity and Inclusion" or DEI.  The goal of the call was to have an open discussion about steps US-RSE might take in promoting diversity within the US-RSE organization and externally in our profession. A blog post summarizing the call can be found [here](https://us-rse.org/2020-10-13-community-call-dei/). As a result of the call, a new DEI working group was formed along with a slack channel, [#dei-discussion](https://usrse.slack.com/archives/C01C8CJQ7AP), for related discussions. The DEI working group has met once since the original community call and is now scheduling a second meeting. All are welcome to join. Because of the [AGM](https://us-rse.org/events/2020/2020-12-AGM) in December, the next community call will take place in January. Those interested in proposing a topic for any of the future community calls should do so in the [#communitycalls](https://usrse.slack.com/archives/CLBDQMJH5) Slack channel.
+
+<a name="leaders"></a>
+# **The 2nd International Leaders Workshop**
+
+The [2nd International Leaders Workshop](https://researchsoftware.org/2020-workshop.html) took place in September and attracted 39 participants from 13 countries on five continents, and included delegates from eight national or multinational RSE associations: [*de-RSE*](https://de-rse.org/en/), [*Society of RSE (UK)*](https://society-rse.org/), [*NL-RSE*](https://nl-rse.org/), [*US-RSE*](https://us-rse.org/), [*Nordic RSE*](https://nordic-rse.org/), [*African RSSE*](https://rsse-africa.sanbi.ac.za/login), [*beRSE*](https://www.be-rse.org/), and [*AUS/NZ RSE*](https://rse-aunz.github.io/). Even though they do not have formal RSE associations, individuals from Argentina, Colombia, France, Canada, and Spain also joined the workshop. The international attendance is thrilling and we are looking forward to more international collaboration! You can find more information about the workshop in the blog posts linked below in Interesting Reads.
```

Comment:
I don't understand what "The international attendance is thrilling" means, given this was an international leaders workshop

Reply:
I think it's meant to say that, for most of us that don't typically interact with people from so many different places, it's thrilling to do that.

Reply:
Maybe we can say that instead?

Reply:
I interpreted it to mean, it was thrilling to see such broadly international attendance.

Reply:
I think @sandragesing wrote it, if she wants to weigh in.

Reply:
> it was thrilling to see such broadly international attendance

Can we say that then?

Reply:
Ok. I changed it.

Reply:
Yeah - that's how I meant it. The change is fine with me